kubespawner/objects.py

```diff
@@ -169,31 +171,35 @@ def make_pod(
         https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/.
 
         Pass this field an array of "Toleration" objects.*
-        * https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#nodeselectorterm-v1-core
+        #nodeselectorterm-v1-core
```

Comment:
Looks like whatever auto-wrapped these lines mangled the URLs :/

Reply:
Demangled.  I think I got them all?